README.md

```diff
@@ -10,12 +10,17 @@ This is the best guide I have ever seen.
 
 ## How to preview the changes
 
-Bundle the dir and then load in the settings.
+1. Generate the bundle file
 
 ```sh
-tar -czf discourse-theme-asktug.tar.gz discourse-theme-asktug
+npm run tar
 ```
 
+2. Upload the generated bundle from the admin settings
+
+- Test Evn: https://idc.asktug.com/admin/customize/themes
```

Comment:
Typo found.

Reply:
Good catch, will update soon.

Reply:
Done. Please check my fix at https://github.com/tidb-incubator/discourse-theme-asktug/pull/3/commits/88ca3dcc8b54e6b557590a3528b7d79384165d11#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Reply:
Also the reason why I prefer camelCase was raised at https://github.com/tidb-incubator/discourse-theme-asktug/pull/2#discussion_r587258210

> The relative path help removing conflicts/uncertainties if mix-using path alias/mappings.
Camel case is optimised for the global search comparing with snake_case. For example, when you search the styling file, it may save a character for _.